.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,4 @@ Make sure that the following steps are done before merging
   - [ ] are you adding any new default values that need to be overridden when this goes live?
```

Comment:
@xirdneh, this already conveys what we want. Can we use it as-is?

Reply:
I think we could just keep that one. I thought we wanted something more explicit.